wdtk-client/src/main/java/org/wikidata/wdtk/client/Client.java

```diff
@@ -204,6 +214,32 @@ private void openActions() {
 	}
 
 	/**
+	 * Creates a report file including the results which came from the
+	 * {@link DumpProcessingAction#getReport()} methods or printing it to
+	 * stdout.
+	 * 
+	 * @throws IOException
+	 */
+	void createReport() throws IOException {
+		StringBuilder builder = new StringBuilder();
+		for (DumpProcessingAction action : this.clientConfiguration
+				.getActions()) {
+			builder.append(action.getActionName() + ": ");
+			builder.append(action.getReport());
+			builder.append(System.getProperty("line.separator"));
+		}
+		if (this.clientConfiguration.reportFilename != null) {
+			File reportFile = new File(this.clientConfiguration.reportFilename);
+			BufferedWriter writer = new BufferedWriter(new FileWriter(
+					reportFile));
+			writer.write(builder.toString());
+			writer.close();
+		} else {
+			logger.info((builder.toString()));
```

Comment:
Wouldn't it be nicer to have one info log message per action instead of having each of them in one report?